website/data/docs-nav-data.json

```diff
@@ -1251,8 +1251,25 @@
         "path": "agent/config-entries"
       },
       {
-        "title": "Telemetry",
-        "path": "agent/telemetry"
+        "title": "Monitor Consul",
+        "routes": [
+          {
+            "title": "Agent telemetry",
+            "path": "agent/monitor/telemetry"
+          },
+          {
+            "title": "Monitor components",
+            "path": "agent/monitor/components"
+          },
+          {
+            "title": "Monitor service communication",
+            "path": "agent/monitor/service"
```

Comment:
I don't think this is an intuitive place for it if we're centering around jobs to be done, but ok if this is a temp measure

These are usually two different personas:
- Consul operator monitoring Consul itself
- Service operator/developer monitoring the services on Consul

Reply:
This is a temp place, with the only req from me is that they all go in the same location (for now anyway).

Reply:
If it's easier to discover, splitting into two locations is probably fine, especially since we will have redirects sending people who bookmarked it during the first publication.